### ISSUE TITLE
Validation Flash Message (accessibility)

### ISSUE THREAD

**Issue by jonibar** [psu-stewardship/etda#89]
Validation flash message is not clickable.  After validation, cursor should be positioned at form field; not validation message.

Think about moving flash messages from header to top of form so that they're closer to where the error occurred.

**Comment by cam156** [comment]
Superseded by #126
